fakeserver/server.go

```diff
@@ -64,7 +57,23 @@ func Start(listenOn string) {
 			}
 			return false
 		},
-	}).Handler(r)))
+	}).Handler(next)
+}
+
+// Start the server
+func Start(listenOn string, logRequests bool) {
+	r := mux.NewRouter()
+
+	s := &server{router: r}
+	s.routes()
+
+	if logRequests {
+		r.Use(loggingMiddleware)
+	}
+	r.Use(corsMiddleware)
+
+	fmt.Printf("testtrack server listening on %s\n", listenOn)
```

Comment:
We might want to make this a standard log message if we turn logging on all the time.